pkg/channel/consolidated/reconciler/controller/resources/dispatcher.go

```diff
@@ -112,7 +112,7 @@ func makeEnv(args DispatcherArgs) []corev1.EnvVar {
 		Value: "config-logging",
 	}, {
 		Name:  "CONFIG_LEADERELECTION_NAME",
-		Value: "config-leader-election-kafka",
+		Value: "config-leader-election",
```

Comment:
Don't you want to read that from the env?

Reply:
yeah, this here "makes" the dispatcher ...

Reply:
But do we want to force it to use the 'global' config? Wouldn't it be wiser to pass whatever the controller is using? (which I guess is a less surprising behaviour)

Reply:
> But do we want to force it to use the 'global' config? Wouldn't it be wiser to pass whatever the controller is configured to use? (which I guess is a less surprising behaviour)

I'd like to address this later - it's generally funky :-) 
This is only executed if the deployment of the dispatcher needs updates...  Perhaps it's cleaner to ignore them in code?
but than it would be a diff the actual deployment cfg we have on file